tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+- name: remove existing mariadb install
+  yum: pkg={{item}} state=absent
+  with_items:
+   - MariaDB
+   - MariaDB-server
+   - MariaDB-libs
+
+- name: Add MariaDB repo
+  template: src=mariadb_centos.repo.j2 dest=/etc/yum.repos.d/mariadb.repo owner=root group=root mode=0644
```

Comment:
- Break up as in yum above
- Ensure all variables are quoted *except* integers or booleans